### PR TITLE
Create major_change issue template

### PR DESCRIPTION
**Pull request by oli-obk**
As per the proposed process from https://github.com/rust-lang/compiler-team/blob/master/content/minutes/design-meeting/2019-12-20-major-change-process.md#final-proposed-process

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+assignees: ''
+
+---
+
+# Directions
+
+If you'd like to propose a major change to do to rustc, you've come to
+the right place! To do so, please write-up your proposal in this issue.
+
+Describe your major change in a sentence or two under the TLDR section.
```
Comment (spastorino) [inline .github/ISSUE_TEMPLATE/major_change.md]:
```suggestion
Describe your major change in a sentence or two under the TL;DR section.
```

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Just add any information you think is important.
+
+List people already mentoring you on this change or reviewing this change in the "Mentors or Reviewers" section
+If there are none, you can also add people that you think would be a good fit.
+You can look at [the experts map](https://github.com/rust-lang/compiler-team/blob/master/content/experts/map.toml) for ideas.
+
+Oh, and please delete this section before you open the issue -- but
+keep the others!
+
+# TLDR
```
Comment (spastorino) [inline .github/ISSUE_TEMPLATE/major_change.md]:
```suggestion
# TL;DR
```

**Comment by spastorino** [comment]
cc @nikomatsakis for final decision

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+name: Major change announcement
+about: Propose a major change.
+title: "(My major change proposal)"
+labels: major-change
+assignees: ''
+
+---
+
+# Directions
```
Comment (bjorn3) [inline .github/ISSUE_TEMPLATE/major_change.md]:
```suggestion
<!--

# Directions
```

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+If available, add links and more descriptions in the "Links and Details" section.
+Note: it is not expected that you write a fully fleshed out proposal there.
+Just add any information you think is important.
+
+List people already mentoring you on this change or reviewing this change in the "Mentors or Reviewers" section
+If there are none, you can also add people that you think would be a good fit.
+You can look at [the experts map](https://github.com/rust-lang/compiler-team/blob/master/content/experts/map.toml) for ideas.
+
+Oh, and please delete this section before you open the issue -- but
+keep the others!
```
Comment (bjorn3) [inline .github/ISSUE_TEMPLATE/major_change.md]:
```suggestion
-->
```

**Comment by nikomatsakis** [comment]
cc @michaelwoerister

**Comment by nikomatsakis** [comment]
Maybe we want to add a section to link to specific Zulip topic(s) where discussion is taking place?

**Comment by nikomatsakis** [comment]
Also, I wonder if they should be in a new stream, `#t-compiler/major-changes`

**Comment by nikomatsakis** [comment]
I at least find #t-compiler a bit..."full"

**Comment by spastorino** [comment]
> I at least find #t-compiler a bit..."full"

I have the feeling that as much as we can group things from #t-compiler into #t-compiler/category the better.

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+about: Propose a major change.
+title: "(My major change proposal)"
+labels: major-change
+assignees: ''
+
+---
+
+# Directions
+
+If you'd like to propose a major change to do to rustc, you've come to
```
Comment (michaelwoerister) [inline .github/ISSUE_TEMPLATE/major_change.md]: If you'd like to propose a major change ~~to do~~ to rustc

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+keep the others!
+
+# TL;DR
+
+# Links and Details
+
+# Mentors or Reviewers
+
+* Chuck Norris
+* Obi Wan Kenobi
```
Comment (michaelwoerister) [inline .github/ISSUE_TEMPLATE/major_change.md]: While this list makes me chuckle personally, it might be a good idea to set a different tone here and steer clear of what could be perceived as brogrammer humor. It's a template text that people will regular have on their screens, it should try to be mostly neutral, I think.

File: .github/ISSUE_TEMPLATE/major_change.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+title: "(My major change proposal)"
+labels: major-change
+assignees: ''
+
+---
+
+# Directions
+
+If you'd like to propose a major change to do to rustc, you've come to
+the right place! To do so, please write-up your proposal in this issue.
```
Comment (nagisa) [inline .github/ISSUE_TEMPLATE/major_change.md]: `s/in this issue/using this template/`

File: .github/ISSUE_TEMPLATE/major_change.md (hunk 4 above)
Comment (bjorn3) [reply .github/ISSUE_TEMPLATE/major_change.md]: The "please" delete can be removed, as comments don't show up in the rendered view.

**Comment by nikomatsakis** [comment]
I'm going to merge this, and we may make a few tweaks as we finalize the RFC
